src/components/ads/AdHandling.js

```diff
@@ -116,8 +116,8 @@ const AdHandler = () => {
 
 
     return (
-        <div className="pt-10">
-           <div className="container pt-4 pb-4 w-11/12 min-h-screen m-auto flex flex-col">
+        <div className={`pt-10 ${selectedAd ? 'hidden-navbar' : ''}`}>
```

Comment:
```suggestion
        <div className=pt-10>
```

Reply:
Yeah it is not needed

Reply:
because does not seem to be necessary here